src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7574,6 +7574,63 @@ public enum NormalizationForm
         FormKC = 5,
         FormKD = 6,
     }
+    public readonly struct Rune : IComparable<Rune>, IEquatable<Rune>
```

Comment:
See if you can use the ref generation tool to auto-gen the reference so it has correct sort ordering and fully qualified type names (like `System.IEquatable<Rune>`).

`msbuild /t:GenerateReferenceSource`

However, for System.Runtime, there are some warts with the auto-gen tool, so you may have to undo some of the changes that it makes (to other types).

Reply:
There's no documentation for how to run that tool or what parameters to give it. If somebody creates docs I can run it. Otherwise I'll stick with doing this by hand because at least I know by hand works. :)

Reply:
> There's no documentation for how to run that tool or what parameters to give it. 

Agreed. Here is how I tend to do it (for System.Runtime):

1) Build coreclr release
2) Build corefx release with coreclr bits
3) Run `msbuild /t:GenerateReferenceSource /p:ConfigurationGroup=Release` from the System.Runtime/ref directory
4) Filter out all unrelated changes and extract the changes you care about (ignore certain attributes being removed).

https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md#testing-with-private-coreclr-bits

Updated the ref here: https://github.com/dotnet/corefx/pull/33512

Reply:
Added documentation here: https://github.com/dotnet/corefx/pull/33515